src/NServiceBus.Testing/Handler.cs

```diff
@@ -4,24 +4,25 @@
     using System.Collections.Generic;
     using System.Linq;
     using System.Reflection;
+    using NServiceBus.MessageInterfaces.MessageMapper.Reflection;
+    using NServiceBus.Testing.ExpectedInvocations;
 
     /// <summary>
     /// Message handler unit testing framework.
     /// </summary>
     public class Handler<T>
     {
-        private readonly StubBus bus;
         private readonly T handler;
-        private IDictionary<string, string> incomingHeaders = new Dictionary<string, string>();
-        private IList<IExpectedInvocation> expectedInvocations = new List<IExpectedInvocation>();
+        IMessageCreator messageCreator = new MessageMapper();
```

Comment:
We could but I believe @hmemcpy explicitly did it this way to avoid creating two instances?